interop-testing/build.gradle

```diff
@@ -34,6 +34,7 @@ dependencies {
     compileOnly libraries.javax_annotation
     runtimeOnly libraries.opencensus_impl,
             libraries.netty_tcnative,
+            libraries.netty_epoll,
```

Comment:
This should not be necessary; ALTS is using grpc-netty-shaded which includes epoll, but also _can't_ use this epoll dep.

Reply:
removed